security/medusa/Kconfig

```diff
@@ -80,15 +80,18 @@ config SECURITY_MEDUSA_FUCK_HASH_TABLE_SIZE
 	depends on SECURITY_MEDUSA
 	default 3
 	help
-	  Defines the number of buckets a FUCK hash table stored in
+	  Defines the number of buckets as 2^n a FUCK hash table stored in
```

Comment:
Exponent used to compute the number of buckets (2^n} of a FUCK hash table…

Reply:
Determines the number of hash table buckets (exponent)

Reply:
Should be fixed now